zulipterminal/model.py

```diff
@@ -1567,6 +1568,13 @@ def _handle_message_event(self, event: Event) -> None:
 
             self.controller.update_screen()
 
+    def _handle_alert_words_event(self, event: Event) -> None:
```

Comment:
Is there a better location for this function to be added? :)

I know model.py is long, but the function above uses the function below, it seems.

Reply:
Moved the position in between `_handle_subscription_event` and `_handle_typing_event`